torchgeo/datasets/geo.py

```diff
@@ -339,7 +339,8 @@ def __init__(
         """Initialize a new Dataset instance.
 
         Args:
-            root: root directory where dataset can be found
+            paths: root directory or list of absolute filepaths where
+                dataset can be found
```

Comment:
Can we change this to something like:

> paths: one or more directories or files to load

This makes it clear that:

* A list of directories is supported
* A single file is supported

The only thing I don't love is idk how to express that some datasets can be auto-downloaded and this is where they will be downloaded to, not where they are found.

This will break support for glob chars in directories, but I'm not sure if anyone was using that anyway.

Reply:
Fixed